corporate/lib/stripe.py

```diff
@@ -633,6 +633,19 @@ def do_change_remote_server_plan_type(remote_server: RemoteZulipServer, plan_typ
     )
 
 
+@transaction.atomic
+def do_deactivate_remote_server(remote_server: RemoteZulipServer) -> None:
+    if not remote_server.deactivated:
```

Comment:
Cleaner would be to do `if remote_server.deactivated: return` as an early return approach.

That said, generally, avoid `do_*` functions that can fail to do something; is it expected that this code path will only happen for an active remote server?  If so, I'd do a `logging.warning()` before returning.